source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -35,69 +35,9 @@ tests:
                   insertedId: 1
         result:
           errorCodeName: WriteConcernFailed
-          # Per transactions spec, drivers add UnknownTransactionCommitResult
-          # label for WriteConcernFailed errors; however, neither the driver
-          # nor withTransaction() will retry the commit in this case.
           errorLabelsContain: ["UnknownTransactionCommitResult"]
           errorLabelsOmit: ["TransientTransactionError"]
-    expectations:
-      -
-        command_started_event:
-          command:
-            insert: *collection_name
-            documents:
-              - { _id: 1 }
-            ordered: true
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            startTransaction: true
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            writeConcern: ~
-          command_name: insert
-          database_name: *database_name
-      -
-        command_started_event:
-          command:
-            commitTransaction: 1
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            startTransaction: ~
-            writeConcern: ~
-          command_name: commitTransaction
-          database_name: admin
-    # The write operation is still applied despite the write concern error
-    outcome: &outcome
-      collection:
-        data:
-          - { _id: 1 }
-  -
-    # This test configures the fail point to return an error with the
-    # WriteConcernFailed code but without errInfo that would identify it as a
-    # wtimeout error. This tests that drivers do not assume that all
-    # WriteConcernFailed errors are due to a replication timeout.
-    description: commitTransaction is retried after WriteConcernFailed non-timeout error
-    failPoint:
-      configureFailPoint: failCommand
-      mode: { times: 2 }
-      data:
-          failCommands: ["commitTransaction"]
-          # Do not specify closeConnection: false, since that would conflict
-          # with writeConcernError (see: SERVER-39292)
-          writeConcernError:
-            code: 64
-            codeName: WriteConcernFailed
-            errmsg: "multiple errors reported"
-    operations:
-      -
-        name: withTransaction
-        object: session0
-        arguments: *arguments
-    expectations:
+    expectations: &expectations_with_retries
```

Comment:
These expectations are the same as those for the first "WriteConcernFailed timeout error" test, so I created an anchor in the first test and used an alias here. The only difference between both tests is the fail point configuration.